triton-server/src/api/download/routes.ts

```diff
@@ -1,13 +1,13 @@
 import express, { Request, Response } from "express"
 import asyncHandler from "express-async-handler"
-import { dataHandler, errorHandler } from "./handlers"
-import { defaultDatabaseActions } from "./actions"
-import { getFreezeManAuthenticatedAPI } from "../freezeman/api"
-import { NewDownloadFile } from "./download-types"
+import { dataHandler, errorHandler } from "@notifications/emails/handlers"
+import { defaultDatabaseActions } from "@database/download/actions"
+import { getFreezeManAuthenticatedAPI } from "@api/freezeman/api"
+import { NewDownloadFile } from "../../types/download"
```

Comment:
Can this be converted to absolute path?

Reply:
unfortunately no
this was a blocker when i was adding the absolute paths for that folder
the gist of problem was because of the the packaged nature of where the imports come from, (.d.ts)
the packaged nature and the use of it to be precise